src/CustomerIOPushMessaging.tsx

```diff
@@ -0,0 +1,33 @@
+import { NativeModules, Platform } from 'react-native';
+
+const LINKING_ERROR =
+  `The package 'customerio-reactnative' doesn't seem to be linked. Make sure: \n\n` +
+  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
+  '- You rebuilt the app after installing the package\n' +
+  '- You are not using Expo managed workflow\n';
+
+/**
+ * Get CustomerIOPushMessaging native module
+ */
+const PushMessagingNative = NativeModules.CustomerioPushMessaging
+  ? NativeModules.CustomerioPushMessaging
+  : new Proxy(
+      {},
+      {
+        get() {
+          throw new Error(LINKING_ERROR);
+        },
+      }
+    );
+
+class CustomerIOPushMessaging {
+  onMessageReceived(message: any): Promise<boolean> {
+    if (Platform.OS === 'ios') {
+      return Promise.resolve(true);
```

Comment:
Could we add a comment explaining why we ignore iOS for now?

Reply:
Will add comments to it. I'm thinking this might not be needed for iOS right now. But that's not yet final. We can discuss more this on [parent ticket](https://github.com/customerio/issues/issues/9591#issuecomment-1477554569).

Reply:
I agree we can discuss in the parent ticket. 

Having a comment in the code is a note to future developers reading this code wondering, "Why is the iOS implementation missing here?".

Reply:
Agreed, added comments to it